api/link_glibclinux_aarch64.go

```diff
@@ -0,0 +1,6 @@
+//go:build linux && !muslc && arm64
+
+package api
+
+// #cgo LDFLAGS: -Wl,-rpath,${SRCDIR} -L${SRCDIR} -lwasmvm.aarch64
```

Comment:
Nice. With this split of link_*.go files we can rename the other one to `-lwasmvm.x86_64` / `libwasmvm.x86_64.so` for consistency.